graphql/requirements.txt

```diff
@@ -4,4 +4,4 @@ fastapi==0.85.0
 pydantic==1.10.2
 SQLAlchemy==1.4.11
 ../shared_libraries[database]
-psycopg2-binary
+psycopg2
```

Comment:
the requirements-dev.txt will still use psycopg2-binary, right? Just making sure.

Reply:
That's my initial impulse. I could build the full psycopg2 for tests if we want, though unit tests shouldn't be running psycopg2 code anyway.